website/docs/r/virtual_network_gateway.html.markdown

```diff
@@ -188,6 +188,7 @@ The `bgp_settings` block supports:
 
 * `peer_weight` - (Optional) The weight added to routes which have been learned
     through BGP peering. Valid values can be between `0` and `100`.
+    Changing this forces a new resource to be created.
```

Comment:
I think this should be for `peering_address` instead of `peer_weight`

Reply:
good spot - fixed.